react/training-project/src/app/screens/Game/components/Board/components/Square/styles.scss

```diff
@@ -1,4 +1,4 @@
-@import '../../../scss/colors';
+@import 'src/scss/_colors';
```

Comment:
Check it, but I think you can add an env variable `NODE_PATH=src/`so you can do `scss/_colors`.